Fix:
error: use of undeclared identifier 'free'
error: use of undeclared identifier 'malloc'
error: use of undeclared identifier 'qsort'
https://github.com/elasota/Aerofoil/commit/53ff18d337e35aaefb866a43f7924e7d182c0f77
--- AerofoilPortable/GpThreadEvent_Cpp11.cpp.orig	2021-11-10 17:05:19.000000000 -0600
+++ AerofoilPortable/GpThreadEvent_Cpp11.cpp	2022-03-17 02:42:47.000000000 -0500
@@ -1,5 +1,7 @@
 #include "GpThreadEvent_Cpp11.h"
 
+#include <stdlib.h>
+
 GpThreadEvent_Cpp11::GpThreadEvent_Cpp11(bool autoReset, bool startSignaled)
 	: m_flag(startSignaled)
 	, m_autoReset(autoReset)
--- AerofoilX/GpLogDriver_X.cpp.orig	2021-11-10 17:05:19.000000000 -0600
+++ AerofoilX/GpLogDriver_X.cpp	2022-03-17 02:35:36.000000000 -0500
@@ -4,6 +4,7 @@
 #include "GpApplicationName.h"
 #include "GpIOStream.h"
 
+#include <stdlib.h>
 #include <time.h>
 #include <cstring>
 
--- PortabilityLayer/ZipFileProxy.cpp.orig	2021-11-10 17:05:19.000000000 -0600
+++ PortabilityLayer/ZipFileProxy.cpp	2022-03-17 02:11:38.000000000 -0500
@@ -10,6 +10,7 @@
 #include "DeflateCodec.h"
 
 #include <algorithm>
+#include <stdlib.h>
 
 namespace
 {
